torchgen/model.py

```diff
@@ -892,6 +892,10 @@ def view_schema_kind(self) -> ViewSchemaKind:
     def root_name(self) -> str:
         return self.func.name.name.base
 
+    @property
+    def part_of_structured_group(self) -> bool:
+        return self.structured or self.structured_delegate is not None
```

Comment:
Should `structured_inherits` be in here too?

Reply:
Also, a nice check for this would be in `NativeFunctionsGroup.__post_init)__` to check that the out=/functional/inplace variants in the group are either all `part_of_structured_group` or all not.

Reply:
> Should `structured_inherits` be in here too?

Not sure. AFAIU, `structured_inherits` implies `structured`, doesn't it?

Reply:
@bdhirsh is this understanding correct?

Reply:
yep!